web_widget_kanban_many2one/static/src/js/web_widget_kanban_many2one.js

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2019 Sunflower IT <www.sunflowerweb.nl>
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+ */
+
+odoo.define('web_widget_kanban_many2one.kanban_editable', function(require){
+
+    var AbstractField = require('web.AbstractField');
+    var core = require('web.core');
+    var field_registry = require('web.field_registry');
+
+    var _t = core._t;
+
+    var many2one_kanban = AbstractField.extend({
+        template: _t('Kanban_editable_temp'),
+        events: _.extend({}, AbstractField.prototype.events, {
+            'click .selectM2MField': function(event){
+                event.stopPropagation();
+            },
+            'change .selectM2MField': '_onSelectField'
+        }),
+        _render: function () {
+            var el = this.$el;
+            var select = el.find('.selectM2MField');
```

Comment:
widgets have '$' function that call to "this.$el.find("...
```suggestion
            var select = this.$('.selectM2MField');
```